### ISSUE TITLE
MacOS: Massive amount of notes cant be viewed because of width, heigth, x and y

### ISSUE THREAD

**Issue by hedgecrog** [tomboy-notes/tomboy-ng#103]
I have a couple of hundreds of notes originally from the original tomboy app imported into my macos tomboy-ng installation.

Almost a hundred of them can't be viewed because of the width, height, x and y settings in them - probably an old default in an old version of tomboy. 

Anyhow - tomboy-ng should of course always write the windows even when these settings are wonky in the notes. See below and try for yourself how tomboy-ng handles the following;

```
  <cursor-position>1</cursor-position>
  <selection-bound-position>1</selection-bound-position>
  <width>0</width>
  <height>0</height>
  <x>-1</x>
  <y>-1</y>
  <open-on-startup>True</open-on-startup>
```

Macos recognizes that the notes are open and somehow written to the screen buffer (try mission control), but they cant be viewed, moved, enlarged, read or in any other way interacted with.

**Comment by davidbannon** [comment]
Thanks Hedgecrog, great bug report !  I have no idea how we could end up with width and height set to zero, but I did notice the same phenomena on my Linux box when I recently opened a quite old note.  Now that I know its a real thing, I will make sure we overrule such silly settings at load time, I already do something about notes that are too big, that happens then a note is written to on a very high res screen, then opened on a laptop. So, easy fix in next release. 
Let me know if you need a a fixed but otherwise slightly unsafe test version in the meantime.
Davo

**Comment by hedgecrog** [comment]
Wow, talk about active development! N/p, and I can live with it until the
next release if it gets fixed by then! In the meantime I can still open
necessary notes by closing tomboy-ng and manually edit the relevant tags in
that note.

Best regards,,

Den lör 21 sep. 2019 01:44davidbannon <notifications@github.com> skrev:

**Comment by davidbannon** [comment]
OK, done that. It now leaves a 50x50 window, upper left if note settings are "silly". I am pretty sure no one would want less than this, but we will have to wait and see !

**Comment by davidbannon** [comment]
Fixed in 0.24
